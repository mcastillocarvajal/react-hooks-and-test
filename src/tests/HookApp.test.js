import React from 'react';
import {shallow} from 'enzyme';
import { HookApp } from '../HookApp';



describe('Test for <HookApp />', () => {

    test('should show component correctly', () => {

        const wrapper = shallow( <HookApp /> );
        expect( wrapper ).toMatchSnapshot();
    })
    
    
})
