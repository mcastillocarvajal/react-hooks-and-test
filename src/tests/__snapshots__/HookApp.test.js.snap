// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for <HookApp /> should show component correctly 1`] = `
<div>
  <h1>
    Hola Mundo
  </h1>
</div>
`;
